loopy/target/pyopencl.py

```diff
@@ -767,10 +960,10 @@ class PyOpenCLCASTBuilder(OpenCLCASTBuilder):
     def function_manglers(self):
         from loopy.library.random123 import random123_function_mangler
         return (
-                super().function_manglers() + [
+                [
                     pyopencl_function_mangler,
                     random123_function_mangler
-                    ])
+                    ] + super().function_manglers())
```

Comment:
Why the reordering?

Reply:
For ex. `abs: Union[complex, real]->real` of pyopencl target must have higher preference than it's super class' `abs`.

Reply:
Ah, thx.